codefori/vscode-ibmi
# Cleanup/connection_manager_api

### Changes

Replaces raw access of `connections` setting and password context with a `ConnectionManager` namespace.

### How to test this PR

1. Ensure the same test cases pass
2. Try out the extension as normal
3. Confirm debug still works
4. When editing connection settings or login settings, ensure the sort button is disabled.

### Checklist
